integration/scripts/integration.sh

```diff
@@ -24,6 +24,44 @@ usage() {
     echo ""
 }
 
+configure_kong() {
+    # Wait for Kong to be really healthy
+    while [ "$(docker exec -it integration-kong-1 kong health | grep healthy)" == "" ] ; do sleep 1 ; done
```

Comment:
I like this waiting for health check to pass.  We should use this model elsewhere as well.  Though perhaps we should have a timeout at some point?

Reply:
also why not `docker-compose exec kong` ? `-1` is not always correct...